contact.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Contact us - If Else Network</title>
+		<link rel="stylesheet" href="main.css?v=1" />
+		<script>
+			/* Sanity check for local development */
+			if (window.location.href.match("http") && window.location.host.split(':')[0] !== '127.0.0.1' && window.location.host.indexOf('file:///') < 0) {
+				if (!window.location.href.match("https")) {
+					window.location.href = window.location.href.replace(
+						"http",
+						"https"
+					);
+				}
+			}
+		</script>
+
+		<!-- Social media cards -->
+		<meta name="twitter:card" content="summary" />
+		<meta name="twitter:site" content="@ifelsenetwork" />
+		<meta
+			name="twitter:title"
+			content="Contact the  If Else Network staff!"
+		/>
+		<meta
+			name="twitter:description"
+			content="The IfElse Network empowers the IT industry, especially its younger members, with communication and productivity tools."
+		/>
+		<meta
+			name="twitter:image"
+			content="assets/IfElseLogo.png"
```

Comment:
All of these assets are using paths relative to the location of this file, which is going to be moved into another directory, so a leading `/` should be added to all of the asset paths.

Reply:
E.g. `assets/IfElseLogo.png` becomes `/assets/IfElseLogo.png`